src/views/Teams/Teams.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import { fetchTeams } from '../../services/teams';
+import { Link } from 'react-router-dom';
+
+export default function Teams() {
+  const [teams, setTeams] = useState([]);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const teamsData = await fetchTeams();
+        setTeams(teamsData);
+      } catch (e) {
+        setError(e.message);
+      }
```

Comment:
Great use of try/catch throughout your app! Excellent for error handling with our third party API requests!